mage_ai/streaming/sources/influxdb.py

```diff
@@ -0,0 +1,183 @@
+import datetime
+import time
+from dataclasses import dataclass
+from typing import Callable
+
+from influxdb_client import InfluxDBClient, QueryApi
+from influxdb_client.client.flux_table import TableList
+
+from mage_ai.shared.config import BaseConfig
+from mage_ai.streaming.constants import DEFAULT_BATCH_SIZE
+from mage_ai.streaming.sources.base import BaseSource
+
+
+def build_query(bucket: str, start_time: float, stop_time: float, time_delay: str):
+    '''
```

Comment:
nit: we use google docstring format https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html